modules/@apostrophecms/log/index.js

```diff
@@ -108,16 +108,41 @@ module.exports = {
     self.initFilters();
   },
   methods(self) {
+    // Keep those inside the methods because of performance.
+    // Do not move to the root because of tests.
     const isProduction = process.env.NODE_ENV === 'production';
+    const isTest = self.options.apos?.options?.test;
+    const utilInspect = require('node:util').inspect;
+
+    const formatDevObj = isTest
+      ? (obj) => JSON.stringify(obj, null, 2)
+      : (obj) => utilInspect(obj, {
+        depth: null,
+        colors: true
+      });
     const formatObj = isProduction
       ? JSON.stringify
-      : (obj) => JSON.stringify(obj, null, 2);
-
+      : formatDevObj;
     const formatString = !isProduction
       ? (str, args) => (args.length > 1) ? str.trim() + '\n' : str
       : (str) => str;
 
     return {
+      // Stringify object arguments. If the environment is not `production`,
```

Comment:
If `NODE_ENV` is not production.

Want to distinguish between exactly that versus some notion of distinct deployed environment names, which isn't what `NODE_ENV` does (it's really just a binary switch because so much code in Express etc. looks specifically for `production`).